### ISSUE TITLE
Invalid backend response. After update to 2.1.30

### ISSUE THREAD

**Issue by a2exfr** [Studio-42/elFinder#2324]
I have updated 2.1.28 to 2.1.30 And got this message on upload file.
```
Invalid backend response.
Data is empty.
```

I have rename in connector:
```
		$opts = [
			'bind' => [
				'upload' => [$this,'RenameFiles'], ]
```
and 
```
	function RenameFiles($cmd, $result, $args, $elfinder){
		$files = $result['added'];
		foreach ($files as $i => $file) {
			$parts = pathinfo($file['name']);
			$filename = $this->slug($parts['filename']) . '.' . $parts['extension'];
			$arg = [
				'target' => $file['hash'],
				'name' => $filename];
			$elfinder->exec('rename', $arg);
		}
		return true;
	}
```
I figured that this is the reason of such responce, but in 2.1.28 it working

**Comment by nao-pon** [comment]
@a2exfr Please try add 5th param.
```php
function RenameFiles($cmd, $result, $args, $elfinder, $volume){
```

However, I think that using the 'upload.presave' hook point will give better results. In that way [Normalizer plugin](https://github.com/Studio-42/elFinder/blob/master/php/plugins/Normalizer/plugin.php) is helpful.

**Comment by a2exfr** [comment]
Thanks for your help.
I set debug=>true  and saw that this response caused by my type error, I fix it and it gone.

But, my function RenameFiles still do not work with 2.1.30, even with added 5th params. 
Now I got, this error in console 
`TypeError: files is null  elfinder.full.js:5482:1`

I'm doing transliteration file names to latin and some custom replasment, may be, it can be done in another way(may be Sanitizer), but roll back to old version makes thing work, so may be some changes in new version with 
`$elfinder->exec('rename', $arg);`
I track it work till that moment, and makes correct $arg.
On upoload.presave I using AutoResize plugin but it do not affect on this.
